core/domain/subscription_services.py

```diff
@@ -105,10 +117,13 @@ def unsubscribe_from_creator(user_id, creator_id):
 
 
 def get_all_threads_subscribed_to(user_id):
-    """Returns a list with ids of all the feedback and suggestion threads to
-    which the user is subscribed.
-
-    Callers of this function should ensure that the user_id is valid.
+    """
```

Comment:
Every docstring should have a short summary of the function at the top. Ditto below.

Reply:
Done!